src/common/gpx.h

```diff
@@ -55,6 +61,21 @@ GList *dt_gpx_get_trkseg(struct dt_gpx_t *gpx);
 // get the list of track points for a track segment
 GList *dt_gpx_get_trkpts(struct dt_gpx_t *gpx, const guint segid);
 
+
+void dt_gpx_geodesic_distance(double lat1, double lon1,
+                              double lat2, double lon2,
+                              double *d, double *delta
+                             );
```

Comment:
should be above.

Please add some comments.

Reply:
>should be above

what do you mean? at the beginning of the file?